_posts/en/newsletters/2022-09-14-newsletter.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Bitcoin Optech Newsletter #217'
+permalink: /en/newsletters/2022/09/14/
+name: 2022-09-14-newsletter
+slug: 2022-09-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes our regular section with the summary of
+a Bitcoin Core PR Review Club meeting, a list of new software releases and
+release candidates, and summaries of notable changes to popular Bitcoin
+infrastructure projects.
+
+## News
+
+*No significant news this week.*
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Reduce bandwidth during initial headers sync when a block is found][review club 25720]
+is a PR by Suhas Daftuar that reduces a node's network bandwidth
+requirements while synchronizing the blockchain with peers, including
+during Initial Block Download (IBD). An important part of the Bitcoin
```

Comment:
(1) IIUC, the additional peers logic is only for when we're not caught up and not close to tip?
(2) I think "usage" is better than "requirement." The additional peer would only be added if they announced something.
```suggestion
usage
during Initial Block Download (IBD). An important part of the Bitcoin
```

Reply:
> only for when we're not caught up and not close to tip?

Yes, that's correct, and I initially had some text in there to say that, but it seemed too detailed. The main challenge (for me, anyway) in writing these summaries is where to draw the line between precision and concision (readability).